### PR TITLE
Add r-qtlcharts.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipes/r-qtlcharts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - r-htmlwidgets
+    - r-qtl >=1.30_4
+
+test:
+  commands:
+    - $R -e "library('qtlcharts')"  # [not win]
+    - "\"%R%\" -e \"library('qtlcharts')\""  # [win]
+
+about:
+  home: http
```
Comment (bgruening) [inline recipes/r-qtlcharts/meta.yaml]: url is missing here ...

Comment (bgruening) [reply recipes/r-qtlcharts/meta.yaml]: CI skip commit is enough, thanks @cbrueffer!

**Comment by cbrueffer** [comment]
Gah, fixed, thanks!

**Comment by bgruening** [comment]
Thanks to you! If I have missed something please ping me on all channels :)

**Pull request merged**